queries/tlaplus/highlights.scm

```diff
@@ -57,47 +60,125 @@
   "WF_"
   "WITH"
   "WITNESS"
-  (forall)
+  (address)
+  (all_map_to)
+  (assign)
+  (case_arrow)
+  (case_box)
+  (def_eq)
   (exists)
-  (temporal_forall)
-  (temporal_exists)
-  (set_in)
+  (forall)
   (gets)
-  (case_box)
-  (case_arrow)
   (label_as)
+  (maps_to)
+  (set_in)
+  (temporal_exists)
+  (temporal_forall)
 ] @keyword
+;  Pluscal keywords
 [
-  (def_eq)
-  (maps_to)
-  (all_map_to)
-] @keyword.function
+  (pcal_algorithm_start)
+  "assert"
+  "await"
+  "begin"
+  "call"
+  "define"
+  "end"
+  "fair"
+  "goto"
+  "macro"
+  "or"
```

Comment:
maybe `@keyword.operator`?

Reply:
If you mean "or", it's not a logical operator but a part of the PlusCal `either` clause, e.g.:
```tlaplus
(* --algorithm Test
begin
  either print "a"
  or print "b"
  or print "c"
end either ;
end algorithm *)

```